libs/services/designer-client-services/src/lib/base/function.ts

```diff
@@ -53,4 +54,120 @@ export class BaseFunctionService implements IFunctionService {
     });
     return keysResponse?.default ?? 'NotFound';
   }
+
+  public async fetchSwaggerUrl(functionAppId: string) {
```

Comment:
Can we just expose fetchFuntionAppSwagger and let the swaggerparser be returned in customSwagger so we dont have to do two separate calls in teh caller?